js/dataManager.js

```diff
@@ -26,13 +26,34 @@ export class DataManager{
           acc[counter].question = question;
           acc[counter].answer = answer;
           acc[counter].value = pointValue;
-          acc[counter].category = currentCategory;//categories are camelcase
+          acc[counter].category = this.parseTitle(currentCategory);//categories are camelcase
          
           counter++
         }
       
        return acc;
        }, { })
       }
+
+    parseTitle(str) {
+
+      var strToArray = str.split('');
+      var foundCapitals = [];
+      
+      strToArray.forEach((char) => {
+          if (char === char.toUpperCase()) {
+              foundCapitals.push(strToArray.indexOf(char));
+          }
+      });
+
+      foundCapitals.forEach((capital, i) => {
+          let notMeantToBeCaps = (foundCapitals.indexOf(capital-1) === -1);
```

Comment:
Can we relook at the naming of `notMeantToBeCaps`. This is possibly confusing. Not clear what is happening here.

Reply:
Sure, let me look at it and fix it. 
@JustinD85 
Can you look at it again when I am done?